portfolio/src/main/webapp/script.js

```diff
@@ -39,15 +39,34 @@ function addRandomQuote() {
  * Fetches data and handles response by converting to text for the homepage
  */
 function getDataHomepage() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('quote-container').innerText = quote;
+  fetch('/data').then(response => response.text()).then((messages) => {
```

Comment:
You want to have a consistent style in your files.
For single-argument lambda expressions the parentheses are optional. So it doesn't matter which style you adopt, but it is better to always use the same convention of using "var" or "(var)". Your choice.